maven-plugin/src/main/java/org/openmrs/maven/plugins/GenerateDistro.java

```diff
@@ -0,0 +1,95 @@
+        options.add(O2_DISTRIBUTION);
+        options.add(O3_DISTRIBUTION);
+        String choice = wizard.promptForMissingValueWithOptions(GENERATE_DISTRO_PROMPT, null, null, options);
+
+        DistroProperties distroProperties;
+
+        Server server = new Server.ServerBuilder().build();
+
+        try {
+            server.setServerDirectory(Files.createTempDirectory("distro").toFile());
+        } catch (IOException e) {
+            throw new RuntimeException(e);
+        }
+
+        switch (choice) {
+            case O2_DISTRIBUTION:
+                wizard.promptForRefAppVersionIfMissing(server, versionsHelper, GET_VERSION_PROMPT);
+                if (DistroHelper.isRefapp2_3_1orLower(server.getDistroArtifactId(), server.getVersion())) {
+                    distroProperties = new DistroProperties(server.getVersion());
+                } else {
+                    distroProperties = distroHelper.downloadDistroProperties(new File(outputLocation), server);
+                }
+                distroProperties.saveTo(new File(outputLocation));
+                break;
+
+            case O3_DISTRIBUTION:
+                wizard.promptForO3RefAppVersionIfMissing(server, versionsHelper, GET_VERSION_PROMPT);
+                Artifact artifact = new Artifact(server.getDistroArtifactId(), server.getVersion(),
+                        server.getDistroGroupId(), "zip");
+                Properties frontendProperties;
+                if (new Version(server.getVersion()).higher(new Version("3.0.0-beta.16"))) {
+                    frontendProperties = distroHelper.getFrontendProperties(server);
+                }
+                else {
+                    frontendProperties = PropertiesUtils.getFrontendPropertiesFromSpaConfigUrl(
+                            "https://raw.githubusercontent.com/openmrs/openmrs-distro-referenceapplication/"+ server.getVersion() +"/frontend/spa-build-config.json");
+                }
+                Properties configurationProperties = PropertiesUtils.getConfigurationProperty(artifact);
+                File file = distroHelper.downloadDistro(server.getServerDirectory(), artifact);
+                Properties backendProperties = PropertiesUtils.getDistroProperties(file);
+                Properties spaModuleProperty = PropertiesUtils.getModuleProperty("https://raw.githubusercontent.com/openmrs/openmrs-module-spa/master/pom.xml");
+                Properties allProperties = new Properties();
+                allProperties.putAll(backendProperties);
+                allProperties.putAll(spaModuleProperty);
+                allProperties.putAll(frontendProperties);
+                allProperties.putAll(configurationProperties);
+                distroProperties = new DistroProperties(allProperties);
+                distroProperties.saveTo(new File(outputLocation));
+                break;
+        }
+        logger.info(String.format("openmrs-distro.properties created successfully at %s", outputLocation));
```

Comment:
Can we output the actual location of the file instead of simply a `.`?

Reply:
@dkayiwa I updated the PR.